tensorflow/python/keras/saving/save_test.py

```diff
@@ -37,7 +37,7 @@
 from tensorflow.python.platform import test
 from tensorflow.python.saved_model import loader_impl
 
-if sys.version_info >= (3, 4):
+if sys.version_info >= (3, 6):
```

Comment:
We are still releasing Python3.5 packages. Do we need to increase major to 6?

Reply:
We could also have three different code paths:
1) Python <= 3.4, where there is no (native) pathlib
2) 3.4, 3.5 They have path lib, but do lack the `os.PathLike`, `os.fspath` functions that allow generic implementations of path objects that is not limited to `pathlib.Path`
3) >=3.6 The filesystem path protocol based implementation above

In theory there would be could a fourth one, based on the pypy pathlib backport to python2, but since support is dropping for it I think we don't need that.

Reply:
Can we use the utility function here instead of switching on version?

Reply:
I'm not sure what you mean. This import is there so that the test can construct `pathlib.Path`s to check that the function works. We could move the import into the `test_save_load_tf_pathlib` function, though.

Reply:
Makes sense now, thank you